source/includes/usage-examples/index-code-examples.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+require __DIR__ . '/../vendor/autoload.php';
+
+$uri = getenv('MONGODB_URI') ?: throw new RuntimeException('Set the MONGODB_URI variable to your Atlas URI that connects to the sample dataset');
+$client = new MongoDB\Client($uri);
+
+$database = $client->sample_db;
+$collection = $database->sample_coll;
+
+// start-to-json
+function toJSON(object $document): string
+{
+    return MongoDB\BSON\Document::fromPHP($document)->toRelaxedExtendedJSON();
+}
+// end-to-json
+
+// start-single-field
+$indexName = $collection->createIndex(['<field name>' => 1]);
+// end-single-field
+
+// start-compound
+$indexName = $collection->createIndex(
+    ['<field name 1>' => 1, '<field name 2>' => 1]
+);
+// end-compound
+
+// start-multikey
+$indexName = $collection->createIndex(['<array field name>' => 1]);
+// end-multikey
+
+// start-search-create
+$indexName = $collection->createSearchIndex(
+    ['mappings' => ['dynamic' => true]],
+    ['name' => '<Search index name>']
+);
+// end-search-create
+
+// start-search-list
+foreach ($collection->listSearchIndexes() as $indexInfo) {
+    echo toJSON($indexInfo) . PHP_EOL;
```

Comment:
```suggestion
    echo toJSON($indexInfo), PHP_EOL;
```

See: https://github.com/mongodb/docs-php-library/pull/115#discussion_r1750423119